markdown/index.html

```diff
@@ -63,29 +63,40 @@
           showError("Please grant full access for writing.");
         });
       }
-      grist.ready();
+      grist.ready({
+        columns: [{ name: "Content", type: 'Text'}],
+        requiredAccess: 'full'
+      });
       grist.on('message', (e) => {
         if (e.tableId) { tableId = e.tableId; }
       });
-      grist.onRecord(function(record) {
+      grist.onRecord(function(record, mappings) {
```

Comment:
Is it possible to simplify using `mapColumnNames`?

Reply:
I don't think so. This widget has to update data (so it needs mappings information).
It can be done if we require explicit mapping, but it will break existing widgets - users would need to make the configuration.
What do you think? I'm in favor of breaking existing widgets, as this updates data - hence it's better to require some mapping here.

Reply:
Got it - it is fine as it is then. Let's not break existing usage, but could you add a comment to the code saying that we're supporting an old way of mapping (by hiding all but one column) and new widgets don't need to do this - just so this code doesn't get slavishly copied by others.